web/packages/shared/components/AccessRequests/NewRequest/RequestCheckout/RequestCheckout.tsx

```diff
@@ -250,6 +251,11 @@ export function RequestCheckout<T extends PendingListItem>({
                     key: 'name',
                     headerText: 'Name',
                   },
+                  {
+                    key: 'clusterName',
+                    headerText: 'Cluster Name',
+                    isNonRender: !showClusterNameColumn,
+                  },
```

Comment:
The Web UI shows the cluster name as the first column when you go see details of an access request. Would it make sense to show it as the first column here as well?